### DISCUSSION TITLE
Model attributes not reactive?

### DISCUSSION THREAD

**Discussion by mrFANRA** [CodeDredd/pinia-orm#1178] [General]
Hi

Here is one example:

`const model = useRepo(Model).make();`

```
<input v-model="model.full_name">
<pre>{{ model.full_name }}</pre>
```

In vuex-orm I can see changes in pre tag on input typing. But in pinia-orm nothing happen. Attributes not reactive or it is my mistake somewhere?

`const model = ref(useRepo(Model).make());`
This variant working well, but Im not sure is it good way.

**Comment by CodeDredd** [comment]
Hi,

i am not sure waht your purpose with `make` is. It's ture it is not reactive. It just creates a blank class which isn't persisted. Normally you have a `computed` for the queries to have them reactive.

I think your variant is a valid way to do it for `make`

**Comment by pallabez** [comment]
Would've appreciated an example if anyone can provide for composition api.
